### DISCUSSION TITLE
Linq2Db CLI - any possibility to include secrets.json

### DISCUSSION THREAD

**Discussion by croban** [linq2db/linq2db#3867] [Q&A]
Hi is there any possibility to add secrets.json, to be able to keep connections string "secret"?

**Comment by MaceWindu** [comment]
You mean this https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows ?

No, but I think you can do something in interceptor constructor - it takes options object, which you can modify with data from secrets.json.

I will create feature request to add support.

**Reply by MaceWindu** [reply]
tracked by #3868
